single_session/tests.py

```diff
@@ -45,6 +49,17 @@ def test_login_logout_scenario(self):
         self.logout(self.client, 1)
         self.logout(self.client3)
 
+    @override_settings(SINGLE_USER_SESSION='single_session.tests.validate_user')
+    def test_login_logout_scenario_with_per_user_configuration(self):
+        self.validate_session_number(0)
+        self.login_foo(self.client)
+        self.login_foo(self.client2)  # second session for foo, logs out
+        self.login_bar(self.client, 2)
+        self.login_bar(self.client3, 3)  # second session for bar, stays logged in
+        self.logout(self.client, 1)  # logs out both sessions
+        self.logout(self.client2)
+        self.logout(self.client3)
```

Comment:
It took me a while to work out the number of sessions that should be in place here in this test. I think that I got it correct, but it would be good if you can double-check that the sessions/clients are also correct based on your understanding of the library.